CHANGELOG.md

```diff
@@ -230,6 +230,10 @@ Users/clients that do not make use of these endpoints will not be impacted.
   - Removed ACL setting of private for s3PutObject()
   - Removed ACL confguration on sync-granules task
   - Update documentation on dashboard deployment to exclude ACL public-read setting
+- **CUMULUS-3245**
+  - Update SQS consumer logic to catch ExecutionAlreadyExists error and
+    delete SQS message accordingly.
+  - Add ReportBatchItemFailures to event source mapping
```

Comment:
Can you specify which event source mapping you are adding it too?

Reply:
https://github.com/nasa/cumulus/pull/3389/commits/e395b00c215023cebba918d80722d12673da96bd